src/classes/OPP_OpportunityContactRoles_TDTM.cls

```diff
@@ -383,6 +397,13 @@ public without sharing class OPP_OpportunityContactRoles_TDTM extends TDTM_Runna
                     = OPP_AutomatedSoftCreditsService.createRelationshipOCRs(primaryContactsToOpportunities);
                 dmlWrapper.objectsToInsert.addAll(relationshipOCRs);
             }
+
+            // Create Affiliation Soft Credits accordingly
+            if (!accountsToOpportunities.isEmpty()) {
```

Comment:
same comment as above about trying to optimize our number of soqls against OCR's and avoiding duplicate OCRs.